### PR TITLE
Warnign main ph + all uk to ua

### PR DESCRIPTION
**Pull request by dieie32**
- change all UK to UA
- remove and change the upload button to image-upload-container
- refactor all image upload button to one component - image-upload-container
- change warning position

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "configurations": [
+        {
+            "type": "pwa-chrome",
+            "request": "launch",
+            "name": "Launch Chrome against localhost",
+            "url": "http://localhost:8080",
+            "webRoot": "${workspaceFolder}"
+        }
+    ]
+}
```
Comment (nadiyafomenko) [inline .vscode/launch.json:15]: remove this file from commit, please

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (moran711) [inline .vscode/launch.json:1]: Why do we need this file?

File: debug.log [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[1021/234833.573:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (moran711) [inline debug.log:1]: Please remove this from file and add it to gitignire.

File: src/containers/image-upload-container/image-upload-container.js [hunk 4]
```diff
@@ -1,57 +1,40 @@
...
-          data-cy='add-photo'
-          variant='outlined'
-          color='primary'
-          component='span'
-        >
+        <Button variant='contained' color='primary' component='label'>
           <AttachFile className={styles.attachFile} />
-          Завантажити
+          {buttonLabel}
+          <input
```
Comment (moran711) [inline src/containers/image-upload-container/image-upload-container.js:15]: We have `ImageUploadContainer` for this.

File: src/containers/image-upload-container/image-upload-container.js [hunk 5]
```diff
@@ -1,57 +1,40 @@
...
+            style={{ display: 'none' }}
```
Comment (moran711) [inline src/containers/image-upload-container/image-upload-container.js:16]: Why you set display none here?

File: .vscode/launch.json (hunk 2 above)
Comment (maksym-strus) [reply .vscode/launch.json:1]: we need to move this file to .gitignore

File: cypress/integration/business-pages/business-pages.test.js [hunk 6]
```diff
@@ -60,7 +60,7 @@ describe('Business pages test ', () => {
     cy.get('[data-cy=en]').should('be.visible');
     cy.get('[data-cy=en]').contains('en');
     cy.get('[data-cy=page-header-ua]').should('be.visible');
-    cy.get('[data-cy=page-header-ua]').contains('Заголовок uk');
+    cy.get('[data-cy=page-header-ua]').contains('Заголовок ua');
```
Comment (maksym-strus) [inline cypress/integration/business-pages/business-pages.test.js:63]: Compare this cypress element with appropriate title in config file (it helps us to check whether this title shows right or not

File: cypress/integration/contact.test.js [hunk 7]
```diff
@@ -2,25 +2,25 @@
...
       'Пн: 09:00 - 17:00|Вт: 09:00 - 18:00|Ср: 09:00 - 18:00|Чт: 09:00 - 18:00|Пт: 09:00 - 18:00|Сб: Вихідний|Нд: Вихідний';
     enSchedule =
       'Mon: 9 a.m. - 5 p.m.|Tue: 9 a.m. - 6 p.m.|Wed: 9 a.m. - 6 p.m.|Thu: 9 a.m. - 6 p.m.|Fri: 9 a.m. - 6 p.m.|Sat: Closed|Sun: Closed';
-    ukAddress = 'Львів вул. Угорська, 2';
+    uaAddress = 'Львів вул. Угорська, 2';
     enAddress = 'Lviv 2 Uhorska Str.';
-    ukCartImage =
+    uaCartImage =
       'https://horondi.blob.core.windows.net/horondi/images/thumbnail_26ebx5pkkf8cwuvd_map.png';
     enCartImage =
```
Comment (maksym-strus) [inline cypress/integration/contact.test.js:25]: make sense move these variables to separate file

File: src/containers/image-upload-container/image-upload-container.js (hunk 5 above)
Comment (dieie32) [reply src/containers/image-upload-container/image-upload-container.js:16]: we need to hide standart input

**Review by koropalov — APPROVED** [review]

**Pull request merged**